### ISSUE TITLE
Permission issue when installing

### ISSUE THREAD

**Issue by jakobnissen** [JuliaLang/juliaup#299]
When teaching an introduction to Julia, one of my students couldn't use `juliaup`, and got this message when installing:

```
	name = 'Juliainstaller'
	operating_system = 'unix:OSX'
	crate_version = '1.6.1'
	explanation = '''
	Panic occurred in file 'src/operations.rs' at line 524
	'''
	cause = '''
	called `Result::unwrap()` on an `Err` value: Failed to open juliaup config file.
	​
	Caused by:
	    Permission denied (os error 13)'''
	method = 'Panic'
	backtrace = '''
	​
	   0: 0x102e27642 - juliaup::command_config_modifypath::run_command_config_modifypath::hca3698e1cf4a0e7b
	   1: 0x102d3f314 - juliainstaller::main::hb970c57354254c36
	   2: 0x102d4b286 - std::sys_common::backtrace::__rust_begin_short_backtrace::h89d3ffd42637b2f5
	   3: 0x102d4c12c - _main'''

```
The installation process was run on her own laptop, from a directory where she had write permissions. Unfortunately, I have no way of reproducing it, but perhaps you can use this bug report nonetheless.

**Comment by jakobnissen** [comment]
Okay I got some more information:
* She did not have the file `.zshrc`. On Mac, Juliaup will include this file even if it does not exist
* She does at least now (6 months later) have write access to the directory and all files. None of them are write protected
* The code in Juliaup should work even if the file does not exist. I just did a test on my own computer, and it did not throw an error.

The only explanation I can think of is that MacOs somehow ran juliaup sandboxed or under limited permissions to change some of the bashrc files for security reasons, but this is just me guessing.
So since this can't be reproduced, I'm going to close this.
